pkgs/by-name/ru/rustdesk-flutter/package.nix

```diff
@@ -1,28 +1,31 @@
-{ lib
-, clangStdenv
-, cargo
-, copyDesktopItems
-, fetchFromGitHub
-, flutter316
-, ffmpeg
-, gst_all_1
-, fuse3
-, libXtst
-, libaom
-, libopus
-, libpulseaudio
-, libva
-, libvdpau
-, libvpx
-, libxkbcommon
-, libyuv
-, pam
-, makeDesktopItem
-, rustPlatform
-, rustc
-, rustfmt
-, xdotool
-}: let
+{
```

Comment:
Please separate formatting and code fixing into different commits to make reviewing easier.